### DISCUSSION TITLE
Move the Labels (Tiny Tiny RSS) up.

### DISCUSSION THREAD

**Discussion by VlaK0r** [martinrotter/rssguard#1223] [Q&A] [answered]
![изображение](https://github.com/martinrotter/rssguard/assets/77666575/95e6f91a-b86e-4140-a85a-9aa096d364f0)
Hello!
Thank you for your work
Please tell me, is it possible to move the labels (Tiny Tiny RSS)  to the very top?

**Comment by martinrotter** [comment, marked as answer]
Hi. This sadly not possible (yet). Some items from feed list (labels, regex queries, etc.) are automatically aligned to bottom of the list.
